scripts/getsamplesummary.py

```diff
@@ -46,35 +49,52 @@ def getsampleinfofromname(pars, sample):
       replies = dbc.generalquery( query )
   return replies
 
-def makelinks(samplename, lanedict):
```

Comment:
Just something I like thinking about; a function should preferably do *one* thing. I usually take a step back and think when I make a function like "makelink**s**". Isn't it possible to create a "make_link" function that is called multiple time in a ``for`` loop?

This makes the code more flexible and easier to test ^^